src/OcspBasicResponse.php

```diff
@@ -103,9 +103,14 @@ public function getNextUpdate(): ?DateTime
      */
     public function getSignatureAlgorithm(): string
```

Comment:
A check needs to be added that verifies that the OCSP request and response hash algorithms match.